### ISSUE TITLE
NPCs resorting to silly weapons

### ISSUE THREAD

**Issue by tvm1** [CleverRaven/Cataclysm-DDA#13315]
Leigh Sargent (NPC): `Let's talk, pal!`
Clint "Crowbar" Clinton (ME): `Drop your matfuckin' weapon!`

Leigh Sargent drops S&W 22A and starts to flee.
Leigh Sargent wields a .22 LR.

Now there's zombie dog nearby and Mr. Sargent starts to bash him with .22 LR **ammo**.
He doesn't manages to kill the dog even with help of his trusty ammo and succumbs.

Clint "Crowbar" Clinton proceeds to collect Mr. Sargent's possessions.

Long story short, NPCs should not wield ammo as a weapon.

![screenshot from 2015-08-16 13 11 02](https://cloud.githubusercontent.com/assets/12072674/9293030/f6b03944-4419-11e5-9a48-9fe5f2476cb6.png)

**Comment by chaosvolt** [comment]
Ah yes. I remember when I krava maga'd a gun-toting NPC, disarmed them, and they wielded the ammo. Which I just as easily disarmed.
